deployment/platformsh.rst

```diff
@@ -39,23 +39,25 @@ Platform.sh how to deploy your application (read more about
     name: myphpproject
     
     # The type of the application to build.
-    type: php:5.6
+    type: php:7.1
     build:
-      flavor: symfony
+      flavor: composer
```

Comment:
why do you change this? `symfony` is correct

Reply:
I read somewhere that `composer` is the new and recommended type for all PHP. Their Symfony template doc use that too: https://docs.platform.sh/frameworks/symfony.html

Reply:
good to know, thank you @javiereguiluz 👍